apis/infrastructure/v1beta1/byohost_types.go

```diff
@@ -32,6 +32,26 @@ type ByoHostSpec struct {
 	BootstrapSecret *corev1.ObjectReference `json:"bootstrapSecret,omitempty"`
 }
 
+// HostStatus is a set of details about the host platform.
+type HostStatus struct {
+	// MachineID reported by the host. For unique machine identification
+	// in the cluster this field is preferred. Learn more from man(5)
+	// machine-id: http://man7.org/linux/man-pages/man5/machine-id.5.html
+	HostID string `json:"hostid,omitempty"`
+
+	// The Operating System reported by the host.
+	OSName string `json:"osname,omitempty"`
+
+	// OS Image reported by the host.
+	OSImage string `json:"osimage,omitempty"`
```

Comment:
Can you give an example value for this field?

Reply:
OSName: linux
OSImage: Ubuntu 21.10